Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:19-bullseye
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
```

Comment:
Move the copy instruction to before the `npm install` and remove the previous `COPY` instruction.

Reply:
I did this because of caching in docker, this method helps in caching layers as npm install is the most time-consuming task in making a docker image, we can cache it and only reinstall the node modules if there are any new changes in the package.json file, here the docs - https://docs.docker.com/build/cache/

Reply:
Good catch!